File: .net & mono/Weed3Demo/demo_plus/store/user_get_list.cs
using Noear.Weed;

namespace Weed3Demo.demo_ex {
    //此类，可由工具根据存储过程生成
    //
    public class user_get_list : DbStoredProcedure {
        public user_get_list() : base(DbConfig.test) {
            call("$.user_get_list");
            set("_userID", () => userID);
            set("_sex",() => sex);
        }

        public long userID;
        public int sex;
    }
}

File: java/Weed3Demo/src/main/java/weed3demo/demo_plus/storeProcedure/user_get_list.java
package weed3demo.demo_plus.storeProcedure;

import noear.weed.DbStoredProcedure;
import weed3demo.config.DbConfig;

/**
 * Created by noear on 2017/7/22.
 */
public class user_get_list extends DbStoredProcedure {
    public user_get_list() {
        super(DbConfig.test);

        call("$.user_get_list");
        set("_userID", () -> userID);
        set("_sex",() -> sex);
    }

    public long userID;
    public int sex;
}
